### ISSUE TITLE
Is .NET7 NativeAOT Support wpf, winform and other cross platform UI framework?

### ISSUE THREAD

**Issue by Cricle** [dotnet/runtime#69305]
In .NET Framework can use NGEN `Microsoft.DotNet.Framework.NativeImageCompiler` in wpf, in .NET 6+, can use other to compile wpf to native?

**Comment by Suchiman** [comment]
@Cricle have a look at https://docs.microsoft.com/en-us/dotnet/core/deploying/ready-to-run

**Comment by jkotas** [comment]
Yes, https://docs.microsoft.com/en-us/dotnet/core/deploying/ready-to-run is direct replacement of `Microsoft.DotNet.Framework.NativeImageCompiler`

Check https://github.com/dotnet/runtime/issues/61231 for .NET7 NativeAOT scope. Our NativeAOT target for .NET 7 are console apps. WPF and other similar reflection-heavy frameworks are not in scope for NativeAOT for .NET 7.
